src/libs/ReportUtils.js

```diff
@@ -1852,10 +1852,11 @@ function buildOptimisticIOUReportAction(
  *
  * @param {Object} chatReport
  * @param {Object} iouReport
+ * @param {String} comment - User comment for the IOU.
  *
  * @returns {Object}
  */
-function buildOptimisticReportPreview(chatReport, iouReport) {
+function buildOptimisticReportPreview(chatReport, iouReport, comment) {
```

Comment:
Sorry, we need to make optional param for this function too?

Reply:
Done!